### DISCUSSION TITLE
[Project beta] filter by "no:project" in issue list

### DISCUSSION THREAD

**Discussion by kobakou** [community/community#10749] [Projects and Issues]
I would like to smoothly check the issue which was not arranged in any project.
In this case, I can use issues search term `no:project`. But it does not work with the new beta project.

**Comment by kobakou** [comment]
or supporting these filters is a good solution for me. 
`is:open is:issue -project:REPO/projects/1`

**Reply by thclark** [reply]
Nice workaround, thanks

**Comment by thclark** [comment]
Yes, the new projects really should be included in the issues list filters, it's weird to have issues in projects that appear to be projectless.

**Comment by jwindmuller** [comment]
Finding issues not in a project or not in a specific project is really making hard to use the new projects beta feature

**Comment by kobakou** [comment]
I found today(2022/3/27) that `no:project` filter works well now!!

https://github.com/{ORG}/{REPO}/issues?q=is%3Aissue+is%3Aopen+no%3Aproject
